packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Fixed
+  * Fixed path issue in `terra-folder-tree` docs.
```

Comment:
Isn't this the same line as here? https://github.com/cerner/terra-framework/pull/1994/files#diff-382fcbe2f7c7da08636573af3dcd3cfc1f05427e93c260d61fde217696eed522R17

Reply:
Good catch, thank you for paying attention! Removed in https://github.com/cerner/terra-framework/pull/1994/commits/7121e7831dce695e666b42a7cf1806bf9c5c0081